### DISCUSSION TITLE
How do you add a new section (similar to Docs)

### DISCUSSION THREAD

**Discussion by mrThirsty** [thuliteio/doks#938] [Q&A] [answered]
I have a site up and running and while the Docs section is in use, we want to create another section that works the same as Docs but it is a completely separate  topic than what is currently in the Docs section so would prefer to keep it out of /Docs.

We are looking for the below structure:

/content/en/
   - /Docs/
   - /Arch/

arch being the new root section.

I have tried duplicating the Docs setup and renaming to arch, both structure and in the menus.en.toml and added the new section to the SectionNav param in params.toml but this always causes build errors:

ERROR 2022/11/03 15:49:13 render of "section" failed: "/site/node_modules/@hyas/doks/layouts/_default/list.html:19:13": execute of template failed: template: _default/list.html:19:13: executing "main" at <partial "main/blog-meta.html" .>: error calling partial: "/site/node_modules/@hyas/doks/layouts/partials/main/blog-meta.html:1:17": execute of template failed: template: partials/main/blog-meta.html:1:17: executing "partials/main/blog-meta.html" at <len .Params.contributors>: error calling len: reflect: call of reflect.Value.Type on zero Value
Error: Error building site: failed to render pages: render of "section" failed: "/site/node_modules/@hyas/doks/layouts/_default/list.html:19:13": execute of template failed: template: _default/list.html:19:13: executing "main" at <partial "main/blog-meta.html" .>: error calling partial: "/site/node_modules/@hyas/doks/layouts/partials/main/blog-meta.html:1:17": execute of template failed: template: partials/main/blog-meta.html:1:17: executing "partials/main/blog-meta.html" at <len .Params.contributors>: error calling len: reflect: call of reflect.Value.Type on zero Value
Built in 1136 ms

Is having seperate Docs sections possible, and have I done something wrong, or is it not possible at the moment?

**Comment by h-enk** [comment, marked as answer]
Like so:

```
npm run create -- --kind docs arch
```

See also [Docs based tree](https://getdoks.org/docs/overview/commands/#docs-based-tree)

**Reply by mrThirsty** [reply]
Thanks, not sure how i missed this in the docs.

**Comment by nahidtanvir** [comment]
1. How to use custom weight for arch subdirectory side menu similar to menus.en.toml.

**Reply by h-enk** [reply]
Does this help: https://getdoks.org/docs/how-to/pages-and-menus/menus/#set-order
